pkg/api/v1/pod/util_test.go

```diff
@@ -404,3 +405,140 @@ func TestIsPodAvailable(t *testing.T) {
 		}
 	}
 }
+
+func TestGetContainerStatus(t *testing.T) {
+	type ExpectedStruct struct {
+		status v1.ContainerStatus
+		exists bool
+	}
+
+	tests := []struct {
+		status   []v1.ContainerStatus
+		name     string
+		expected ExpectedStruct
+	}{
+		{
+			status:   []v1.ContainerStatus{{Name: "test1", Ready: false, Image: "image1"}},
```

Comment:
Add another ContainerStatus in the test case to verify that the function retrieves the correct one.